app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: float,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        all_cost = 0
+        if cars:
+            for car in cars:
+                all_cost += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return all_cost
+
+    def calculate_washing_price(self, car: Car) -> int:
+        if car.clean_mark <= self.clean_power:
+            cost_of_washing = round(
+                car.comfort_class * (self.clean_power - car.clean_mark)
+                * self.average_rating / self.distance_from_city_center, 1
+            )
+        else:
+            cost_of_washing = 0
```

Comment:
instead of else declare Cost_of_washing in the first line of the method

Reply:
Can you explain why this is better? Few words pls)